index.js

```diff
@@ -1 +1,94 @@
-console.log('hello world');
+import express from 'express';
+import { read } from './jsonFileStorage.js';
+
+const app = express();
+
+app.get('/recipe/:index', (req, res) => {
+  // get index number from browser req
+  const index = req.params.index;
+  read('data.json', (err, content) => {
+    if (err) {
+      console.log('Read Error', err);
+    }
+    if (content.recipes[index]) {
+      res.send(content.recipes[index]);
+    } else {
+      res.status(404).send('We cannot find that.');
+    }
+  });
+});
+
+/**
+ * returns based on recipe label
+ */
+app.get('/recipe-label/:label', (req, res) => {
+  // get index number from browser req
+  const { label } = req.params;
+  // replace '-' from LABEL to spaces
+  const labelCorrected = label.replaceAll('-', ' ');
+  read('data.json', (err, content) => {
+    if (err) {
+      console.log('Read Error', err);
+    }
+    const recipes = content.recipes;
+    const filteredRecipes = recipes.filter((recipe) => {
+      return recipe['label'].toLowerCase() === labelCorrected;
+    });
+    let output = '';
+    // add the recipe into OUTPUT
+    filteredRecipes.forEach((recipe) => {
+      for (let [key, value] of Object.entries(recipe)) {
+        output += `${key.toUpperCase()}: `;
+        output += `${value}<br>`;
```

Comment:
consider placing it in one line for better readability

```suggestion
        output += `${key.toUpperCase()}: ${value}<br>`;
```